certified-connectors/UiPath Automation Cloud/ReadMe.md

```diff
@@ -0,0 +1,32 @@
+## Author:
+UiPath - [UiPath Website](https://uipath.com)
+
+## Contact:
+UiPath Support Team - [Contact Support](https://www.uipath.com/company/contact-us/contact-technical-support)
+
+## Version: 
+1.0 - Initial Release
+
+## Description
+This is a Power Automate Connector for UiPath Orchestrator.
+With this version only UiPath Automation Cloud is supported.
+
+## Supported Operations
+The connector supports the following operations:
+- [Run Job](https://docs.uipath.com/orchestrator/docs/about-jobs)
+Run a job in UiPath Orchestrator
+- [Add Queue Items](https://docs.uipath.com/orchestrator/docs/about-queues-and-transactions)
+Add a queue item to a specific queue
+
+## Getting Help or Providing Feedback
+If you have any issues, requests for functionality, or have general feedback, please <integrations-apps@uipath.com>.
+
+## Pre-requisites
+Please make sure you have an automation cloud account. [Sign up](https://www.uipath.com/developers/studio-download) for a trial.
+
```

Comment:
Hello @TuckermanRav, 

Please update readme.md file as per the template here and fill all the required fields. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Thanks @vmanoharas.  We've addressed the readme per the template now.

Reply:
@vmanoharas I believe we have resolved all readme issues.